addons/controls/range.js

```diff
@@ -10,9 +10,10 @@
  * * config - config object with at least a `defaultProps` key containing a map of props and their default values
  * * opts - range-specific control options (opts from the standard Controls docs)
  * * preferredValue - (optional) a sample-specific initially selected value
+ * * otherOpts - (optional) other options object unrelated to the control options
```

Comment:
I think it would be better to be more specific
```suggestion
 * * otherOpts - (optional) options specifying argTypes excluding the `control` and `table` keys

Reply:
Thank you, I updated